sustainml_cpp/test/blackbox/common/BlackboxTestsOrchestratorNode.cpp

```diff
@@ -200,9 +200,9 @@ TEST(OrchestratorNode, OrchestratorReceivesNodeOutputs)
 
 TEST(OrchestratorNode, OrchestratorGetTaskData)
 {
-    std::shared_ptr<TestOrchestratorNodeHandle> tonh = std::make_shared<TestOrchestratorNodeHandle>();
+    TestOrchestratorNodeHandle* tonh = new TestOrchestratorNodeHandle();
 
-    orchestrator::OrchestratorNode orchestrator(tonh);
+    orchestrator::OrchestratorNode orchestrator(*tonh);
 
```

Comment:
From now on, user is responsible from deleting the `OrchestratorNodeHandle*`, I encourage you to add at the end of all the tests using the orchestrator (also in examples)
* `orchestrator.destroy()`
*  `delete tonh` 
If you dont want to manually delete the `tonh` you can always make it with `std::make_shared()` and pass the OrchestratorNode constructor its raw pointer `tonh_ptr.get()`